app/app/urls.py

```diff
@@ -214,6 +215,11 @@
     path(settings.SENDGRID_EVENT_HOOK_URL, marketing.webhookviews.process, name='sendgrid_event_process'),
     # gitcoinbot
     url(settings.GITHUB_EVENT_HOOK_URL, gitcoinbot.views.payload, name='payload'),
+    # mentors
+    path('mentor', mentor.views.mentors, name='mentor_list'),
```

Comment:
Mind moving the urls to `mentor/urls.py` and replacing these with something like: `path('mentor/', include('mentor.urls', namespace='mentor')),` ?

Reply:
sure, it's cool, 
just wanted to stick to existing convention.

Reply:
Thanks.  Yeah, moving everything in that direction is on my list, but fairly low priority at the moment.

Much appreciated! 🙌